AutoMaxLair.py

```diff
@@ -90,6 +95,9 @@ def join(inst) -> str:
     inst.pokemon = pokemon_list[selection_index]
     inst.push_buttons((b'a',27))
     inst.log('Choosing a path...')
+
+    inst.caught_pokemon = []

```

Comment:
This line is probably best to put in the MaxLairInstance.reset_run method (you'll have to replace inst with self). That way it will be reset every time a new run is started; furthermore it's best practice to initialize an objects attribute within the object itself.

Reply:
Indeed, I didn't knew the init was a constructor (and thus was calling reset_run :D)